pawsitive/app/views/home/_service_card.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= link_to profile_users_path(service.member.id), target: :_blank do %>
```

Comment:
Hi @jcidp, please change this route to (service_path(service.id)) and remove the tagget _blank because it opens in a new tab.

Reply:
Thanks for the comments @lincoln1883 

Made the changes to redirect to the service and avoid errors.

The only comment I have is that I left the target to open in another tab since that how Rover does it and I think it makes sense so the user can easily checkout multiple members without having to go back between pages.

Let me know what you think.